cupyx/scipy/signal/windows/_windows.py

```diff
@@ -1424,6 +1424,91 @@ def kaiser(M, beta, sym=True):
     return _truncate(w, needs_trunc)
 
 
+def kaiser_bessel_derived(M, beta, sym=True):
```

Comment:
```suggestion
def kaiser_bessel_derived(M, beta, *, sym=True):
```

Reply:
This breaks `get_window` which is not equipped to handle keyword-only arguments. Note that the bulk of the windowing APIs come from cuSignal, and probably predates both introduction of these two windows and the keywod-only stuff

Reply:
I got it. So lets leave the signatures not to have keyword-only parameters in this PR. Thanks!